package.json

```diff
@@ -105,6 +106,7 @@
   },
   "dependencies": {
     "@types/dom-speech-recognition": "^0.0.4",
+    "d3": "^7.9.0",
```

Comment:
Is this needed?  I don't see d3 used in the PR.

Reply:
Thanks for catching that. I've removed it.